quickjs/src/jniTest/kotlin/app/cash/quickjs/ktbridge/KtBridgeTest.kt

```diff
@@ -40,7 +40,7 @@ class KtBridgeTest {
     quickjs.close()
   }
   
-  @Test fun `jvm call js service`() {
+  @Test fun jvmCallJsService() {
```

Comment:
Shall we follow this style for JVM tests? I haven’t been but am happy to

Reply:
Doesn't matter. I personally find it easier to just not use spaces everywhere so that I don't have to worry about where the test is running and whether it supports spaces.